main.tf

```diff
@@ -140,18 +140,12 @@ resource "ibm_is_security_group_rule" "security_group_rule" {
     # Conditianally add content if sg has icmp
     content {
       type = lookup(
-        lookup(
-          each.value,
-          "icmp"
-        ),
+        each.value["icmp"],
```

Comment:
This is just a syntax change right, no other effect on the main module?

Reply:
correct, tflint starting to flag those